portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,29 +34,78 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
   
-  private List<String> facts;
+  private ArrayList<String> comments;
 
   @Override
   public void init() {
-    facts = new ArrayList<>();
-    facts.add(
-        "My favorite color is green. "
-            + "I also like light blue.");
-    facts.add("My parents made up my name based on my Chinese one.");
-    facts.add("My birthday is on Halloween.");
-    facts.add("I have the best younger brother!");
-    facts.add("My favorite food is pasta.");
-    facts.add("I'm a cat person. I like dogs, but I like cats more.");
-    facts.add(
-        "I didn't have any experience with "
-            + "computer science in high school.");
-    facts.add("I play the acoustic guitar.");
+    comments = new ArrayList<>();
   }
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String fact = facts.get((int) (Math.random() * facts.size()));
-    response.setContentType("text/html;");
-    response.getWriter().println(fact);
+    Query query = new Query("Task");
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+
+    // Loop over entities
+    List<Task> tasks = new ArrayList<>();
+    for (Entity entity : results.asIterable()) {
+      // long id = entity.getKey().getId();
+      String name = (String) entity.getProperty("name");
+      String text = (String) entity.getProperty("text");
+      long timestamp = (long) entity.getProperty("timestamp");
+      // comments.add(text);
+      Task task = new Task(name, text, timestamp);
+      tasks.add(task);
```

Comment:
nit: move to a single line like: tasks.add(new Task(name, text, timestamp)) since it's already clear what's happening